sale_order_project/tests/test_sale_order_project.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+import openerp.tests.common as common
+
+
+class TestSaleOrderProject(common.TransactionCase):
+
+    def setUp(self):
+        super(TestSaleOrderProject, self).setUp()
+        self.sale_order_model = self.env['sale.order']
+        self.analytic_account_model = self.env['account.analytic.account']
+        self.sale_order = self.env.ref('sale.sale_order_2')
+        self.project = self.env.ref('project.project_project_1')
+
+    def testCreateProjectFromOrder(self):
```

Comment:
Don't use CamelCase for this method names (there's only the special case of setUp)